pkg/provider/azure_storageaccount.go

```diff
@@ -41,6 +44,7 @@ import (
 const SkipMatchingTag = "skip-matching"
 const LocationGlobal = "global"
 const privateDNSZoneNameFmt = "privatelink.%s.%s"
+const DefaultTokenAudience = "api://AzureADTokenExchange"
```

Comment:
this is a golint error, what's the other token Audience?

Reply:
I don't know any other Audience, for this feature, we only use `api://AzureADTokenExchange` as audience